environments/icds-cas/public.yml

```diff
@@ -117,8 +117,10 @@ couch_dbs:
 TWO_FACTOR_GATEWAY_ENABLED: True
 
 s3_blob_db_enabled: yes
-s3_blob_db_url: "https://mowcdmum.ipstorage.tatacommunications.com"
-s3_blob_db_s3_bucket: 'mowcdmum-c1'
+old_s3_blob_db_url: "https://mowcdmum.ipstorage.tatacommunications.com"
+old_s3_blob_db_s3_bucket: 'mowcdmum-c1'
```

Comment:
how was `old_s3_blob_db_*` defined before?

Reply:
it's defined in `all.yml` as `old_s3_blob_db_s3_bucket: 'blobdb'`